### PR TITLE
Fix installation with online repos resp add-ons

### PR DESCRIPTION
**Pull request by lnussel**
The ordering of wizard steps changed and also some key bindings

### CODE REVIEW

**Comment by okurz** [comment]
And this will work for both TW and SLE? Can you provide a link to a successful test run so that we know how a good run should look like in case it shows problems in master?

File: tests/installation/setup_online_repos.pm [hunk 1]
```diff
@@ -55,7 +61,10 @@ sub run() {
 
     send_key $cmd{next};    # Next
 
-    assert_screen "desktop-selection", 200;    # Make sure repos setup is finished and went to next step already
+    if (get_var("WITH_MAIN_REPO")) {
```
Comment (coolo) [inline tests/installation/setup_online_repos.pm]: yet another variable with a very generic name and a very specific meaning - we urgently need a variable naming convention ;(

**Comment by lnussel** [comment]
there are neither TW nor SLE tests that use addon_products.pm or setup_online_repos.pm

File: tests/installation/setup_online_repos.pm (hunk 1 above)
Comment (lnussel) [inline tests/installation/setup_online_repos.pm]: sure but that variable is not my invention

**Comment by okurz** [comment]
> And this will work for both TW and SLE?

Sorry, was unclear. Which products does this fix? For which one did you implement this?

**Comment by lnussel** [comment]
This is for Leap as I am desperately trying to retrofit openQA on the maintenance update repo there.
A failure looks like this: https://openqa.opensuse.org/tests/125310, a good one like this https://openqa.tanana.suse.de/tests/1490 but that is not 100% accurate as I have quite some more code to make that fly. So I have to cherry pick stuff that is useful to openqa.o.o already.

File: tests/installation/setup_online_repos.pm (hunk 1 above)
Comment (nilxam) [inline tests/installation/setup_online_repos.pm]: it's me! I don't remember why I use those generic name WITH_XXX_REPO, maybe I have not better idea at that moment.

Comment (okurz) [inline tests/installation/setup_online_repos.pm]: As soon as the "need for a variable naming convention" is tracked somewhere else than coolo's brain I would merge it

**Pull request merged**